src/internal/m365/collection/site/restore_test.go

```diff
@@ -187,21 +196,256 @@ func (suite *SharePointRestoreSuite) TestListCollection_Restore_invalidListTempl
 		suite.Run(test.name, func() {
 			t := suite.T()
 
-			lrh, destName, mockData := test.getParams()
+			listData := generateListData(t, service, test.list)
 
 			_, err := restoreListItem(
 				ctx,
 				lrh,
-				mockData,
+				listData,
 				suite.siteID,
-				destName,
+				restoreCfg,
+				nil,
+				count.New(),
 				fault.New(false))
 			require.Error(t, err)
 			assert.Contains(t, err.Error(), api.ErrSkippableListTemplate.Error())
 		})
 	}
 }
 
+func (suite *SharePointRestoreSuite) TestListCollection_RestoreInPlace_skip() {
+	t := suite.T()
+
+	ctx, flush := tester.NewContext(t)
+	defer flush()
+
+	var (
+		listName     = "MockListing"
+		listTemplate = "genericList"
+		restoreCfg   = testdata.DefaultRestoreConfig("")
+		lrh          = NewListsRestoreHandler(suite.siteID, suite.ac.Lists())
+		service      = createTestService(t, suite.creds)
+		list         = createList(listTemplate, listName)
+		newList      = createList(listTemplate, listName)
+		cl           = count.New()
+	)
+
+	mockData := generateListData(t, service, list)
+
+	collisionKeyToItemID := map[string]string{
+		api.ListCollisionKey(newList): "some-list-id",
+	}
+
+	deets, err := restoreListItem(
+		ctx,
+		lrh,
+		mockData,
+		suite.siteID,
+		restoreCfg, // OnCollision is skip by default
+		collisionKeyToItemID,
+		cl,
+		fault.New(true))
+	require.Error(t, err, clues.ToCore(err))
+	assert.True(t, errors.Is(core.ErrAlreadyExists, err))
+	assert.Empty(t, deets)
+	assert.Less(t, int64(0), cl.Get(count.CollisionSkip))
+}
+
+func (suite *SharePointRestoreSuite) TestListCollection_RestoreInPlace_copy() {
```

Comment:
various nits:

Naming structure doesn't follow standards.  It should be `Test<FunctionUnderTest>_<variation>`
```suggestion
func (suite *SharePointRestoreSuite) TestRestoreListItem_collisions() {
```

All four new tests duplicate a lot of boilerplate declarations and behavior.  This bloat makes the tests more difficult to read and, in the long term, will require more effort to maintain and refactor.  They should be condensed into a single test using a test table.

Reply:
@ryanfkeepers 
this change has been accommodated in https://github.com/alcionai/corso/pull/5074